config/endpoint.go

```diff
@@ -37,12 +37,17 @@ func NewEndpointFromCloneURLWithConfig(url string, c *Configuration) Endpoint {
 		return e
 	}
 
+	if strings.HasSuffix(url, "/") {
```

Comment:
Since we already have `path` imported, would it be possible to use `path.Clean` here?